src/domain/session/room/RoomViewModel.js

```diff
@@ -289,10 +278,8 @@ export class RoomViewModel extends ViewModel {
             attachments["info.thumbnail_url"] = 
                 this._room.createAttachment(thumbnail.blob, file.name);
             await this._room.sendEvent("m.room.message", content, attachments);
-        } catch (err) {
-            this._sendError = err;
-            this.emitChange("error");
-            console.error(err.stack);
+        } catch (error) {
+            this.reportError(error);
```

Comment:
So the way I understand it, `reportError()` will eventually lead to a UI showing the error and giving the user the option to submit logs. But I don't see where the error passed to `reportError()` is logged?

Reply:
Indeed, that is how it works. You mean apart from here? I've made a convenience method, `ErrorReportViewModel.logAndCatch`, to combine logging and error handling in one. Perhaps that's why you see `reportError` being called less than expected?